### PR TITLE
Issue#241: Add a field at survey's page for showing how many members respond to survey--Resolved by M.Waqas,Submission Date: May 11, 2016 

### PR DESCRIPTION
**Pull request by MuhammadWaqas05**
https://waffle.io/open-learning-exchange/BeLL-Apps/cards/5722f09fe9d6b90e006bf35a

### CODE REVIEW

**Comment by SadiaRasheed** [comment]
Code is fine and is ready to be merged to main Branch.

File: app/nation/app/views/SurveyTable.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ $(function() {
         },
 
         addAll: function() {
-            this.$el.append('<tr><th>'+App.languageDictValue.get('Date')+'</th><th>'+App.languageDictValue.get("Survey_Number")+'</th><th>'+App.languageDictValue.get('Title')+'</th><th>'+App.languageDictValue.get('Sent_to')+'</th><th>'+App.languageDictValue.get('Submitted_By')+'</th><th>'+App.languageDictValue.get('Response')+'</th><th colspan="2">'+App.languageDictValue.get('Actions')+'</th></tr>')
+            this.$el.append('<tr><th>'+App.languageDictValue.get('Date')+'</th><th>'+App.languageDictValue.get("Survey_Number")+'</th><th>'+App.languageDictValue.get('Title')+'</th><th>'+App.languageDictValue.get('Sent_to')+'</th><th>'+App.languageDictValue.get('Submitted_By')+'</th><th>'+App.languageDictValue.get('Received')+'/'+App.languageDictValue.get('Sent')+'</th><th colspan="2">'+App.languageDictValue.get('Actions')+'</th></tr>')
```
Comment (SabaBaig) [inline app/nation/app/views/SurveyTable.js:20]: Please make sure that newly used words are added in int_docs folder as well; also that whether that word is already there or not. follow https://jsonformatter.curiousconcept.com/ to verify all 8 languages files.

**Comment by sente** [comment]
Should your individual commits be squashed before merging?

If so, you can `git rebase -i dev` to do so.

Thanks!

**Comment by MuhammadWaqas05** [comment]
@sente Yes, we want to squash and will do it.

**Pull request merged**